### ISSUE TITLE
Embed Examples

### ISSUE THREAD

**Issue by naher94** [DzgnIO/toolkit#7]
We should embed examples on the toolkit pages to show what these features look like

**Comment by atopdjian** [comment]
I completely agree. Based on description alone some of them can seem a bit too broad to make someone want to find out more.
